eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -348,6 +348,8 @@ stages:
           timeoutInMinutes: 5
 
         - template: tools/gpg/gpg.yml@azure-sdk-build-tools
+          parameters:
```

Comment:
We should default this in the gpg.yml template given it is in the azure-sdk-build-tools repot.